apps/backend/test/users.e2e-spec.ts

```diff
@@ -377,22 +374,6 @@ describe('/users', () => {
           .expect(HttpStatus.UNAUTHORIZED);
       });
 
-      it('should return 400 status when password does not meet complexity requirements', async () => {
```

Comment:
So I'm fine with removing this test but shouldn't the API return *some* status if password does not meet the requirements? What I mean is we should still have a test for this scenario, right?